forgerock-auth/src/androidTest/java/org/forgerock/android/auth/callback/DeviceBindingCallbackTest.java

```diff
@@ -274,6 +222,58 @@ public void onCallbackReceived(Node node)
         Assert.assertNotNull(FRSession.getCurrentSession());
         Assert.assertNotNull(FRSession.getCurrentSession().getSessionToken());
     }
+
+    @Test
+    public void testDeviceBindingApplicationPin() throws ExecutionException, InterruptedException {
+        final int[] bindSuccess = {0};
+        CallbackFactory.getInstance().register(CustomApplicationPinDeviceBindingCallback.class);
+
+        ActivityScenario scenario = ActivityScenario.launch(DummyActivity.class);
+        scenario.onActivity(InitProvider::setCurrentActivity);
+
+        NodeListenerFuture<FRSession> nodeListenerFuture = new DeviceBindingNodeListener(context, "pin")
+        {
+            NodeListener<FRSession> nodeListener = this;
```

Comment:
This can be final

Reply:
Yeah, this was reported by the linting tool in Android Studio as well... I did fix this one and some other similar cases as well... 
Thanks for pointing it...